scripts/ci/check-hydra.sh

```diff
@@ -1,5 +1,20 @@
-#!/bin/sh
+#!/usr/bin/env nix-shell
+#!nix-shell -p jq -i bash
 
 nix-build https://github.com/nixos/nixpkgs/archive/4fb198892d298452023ab176e7067da58d30772e.tar.gz -A hydra
 echo '~~~ Evaluating release.nix'
-./result/bin/hydra-eval-jobs -I . release.nix
+env time --format '%e' -o eval-time.txt ./result/bin/hydra-eval-jobs -I . release.nix > eval.json
+EVAL_EXIT_CODE="$?"
+EVAL_TIME=$(cat eval-time.txt)
+jq -r '.' < eval.json
```

Comment:
Don't need `-r` (raw output) here.
```suggestion
jq . < eval.json
```